core/plugins/NtupleWriterJets.cxx

```diff
@@ -256,8 +256,8 @@ void NtupleWriterTopJets::process(const edm::Event & event, uhh2::Event & uevent
     }
     vector<TopJet> topjets;
     edm::Handle<edm::View<reco::HTTTopJetTagInfo>> top_jet_infos;
-    event.getByLabel("hepTopTagCHS", top_jet_infos);
-    // Make sure both collections have the same size
```

Comment:
On 07/28/2015 11:16 AM, Thomas Peiffer wrote:
>
> In core/plugins/NtupleWriterJets.cxx
> <https://github.com/UHH2/UHH2/pull/122#discussion_r35628471>:
>
> > @@ -256,8 +256,8 @@ void NtupleWriterTopJets::process(const edm::Event & event, uhh2::Event & uevent
> >      }
> >      vector<TopJet> topjets;
> >      edm::Handle<edm::View<reco::HTTTopJetTagInfo>> top_jet_infos;
> > -    event.getByLabel("hepTopTagCHS", top_jet_infos);
> > -    // Make sure both collections have the same size
>
> Does this change affect the samples that have already been produced?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UHH2/UHH2/pull/122/files#r35628471>.
>
Hi Thomas,
no this should not affect the samples that have already been produced.
Best,
Mareike